cli/src/commands/deploy.ts

```diff
@@ -129,11 +145,17 @@ export default class Deploy extends Command {
     // TODO: select target before this block
     this.startStage('build');
 
+    const stdioInherit = [0, 1, 2];
```

Comment:
nit: This is the default, so if `format === logs`,  you can just avoid passing `spawnOptions` all together (Not a blocker for this PR)